musync_save/main_window.py

```
@@ -587,32 +587,41 @@ def CheckGameRunning(self):
         logger:logging.Logger = Logger.get_logger("MusyncSavDecodeGUI.CheckGameRunning")
         logger.info("Start Thread: CheckGameIsStart.")
         def UpdateUI(text:str, bg:str)->None:
-            self.isGameRunning["text"] = text; #"游戏未启动"
-            self.isGameRunning["bg"] = bg; #"#FF8080"
-        threadInvert:int = 50; # 100ms
-        counter:int = threadInvert
+            self.isGameRunning["text"] = text; #"游戏未启动"
```

Ruff (0.15.9)

[error] Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)